src/main/java/org/ambraproject/rhino/service/impl/ConfigurationReadServiceImpl.java

```diff
@@ -71,6 +90,14 @@
     return cfgMap;
   }
 
+  @Override
+  public ServiceResponse<Map<String, String>> readRunInfo() {
+    Map<String, String> cfgMap = new LinkedHashMap<>(2);
```

Comment:
If `cfgMap` will never be updated, I would suggest using a `com.google.common.collect.ImmutableMap`, rather than a `LinkedHashMap`.  For example:
```
Map<String, String> cfgMap = ImmutableMap.of(
    "host", hostname, "started", startTime.toString());
```

Reply:
I was aiming to make as few changes as possible, but I can use an immutable. Should I change the linked maps for the other responses in this class?

Reply:
Not necessary to modify the code that's not related to the ticket.